api/v1alpha1/constant.go

```diff
@@ -36,3 +36,14 @@ const (
 	// DefaultMode is exclusive
 	DefaultMode PlacementMode = ""
 )
+
+type NetworkMode string
+
+const (
+	HostNetworkMode NetworkMode = "Host"
```

Comment:
I suggest `NetworkMode` is `HostNetwork` and `ContainerNetwork`

Reply:
thx, done